### ISSUE TITLE
aiohttp client doesn't follow redirects of mocked requests

### ISSUE THREAD

**Issue by daltonmatos** [pnuckowski/aioresponses#119]
Hello,

I'm using aioresponses along with aiohttp client and I'm seeing a problem regarding `aiohttp.client.ClientSession` not following redirects of a mocked request. Here are two tests that reproduces the problem:

```python
from aiohttp.client import ClientSession
from aioresponses import aioresponses
from asynctest import TestCase


class AIOResponseRedirectTest(TestCase):
    async def test_aiohttp_follows_mocked_redirect(self):
        with aioresponses() as rsps:
            rsps.get(
                "http://10.1.1.1:8080/redirect",
                status=307,
                headers={"Location": "https://httpbin.org"},
            )
            client = ClientSession()
            response = await client.get(
                "http://10.1.1.1:8080/redirect", allow_redirects=True
            )
            self.assertEqual(200, response.status)

    async def test_aiohttp_follows_passthrough_redirect(self):
        with aioresponses(passthrough=["https://httpbin.org"]) as rsps:
            rsps.get(
                "http://10.1.1.1:8080/redirect",
                status=307,
                headers={"Location": "https://httpbin.org"},
            )
            client = ClientSession()
            response = await client.get(
                "https://httpbin.org/redirect-to?status_code=307&url=https://httpbin.org",
                allow_redirects=True,
            )
            self.assertEqual(200, response.status)

```

What I expected to see is both tests passing, but when I run this testclass, that's what I see:

```
pipenv run py.test -v tests/test_aiohttp_redirect_bug.py
Loading .env environment variables…
===================================================================================== test session starts =====================================================================================
platform linux -- Python 3.6.8, pytest-3.0.1, py-1.8.0, pluggy-0.3.1 -- /home/daltonmatos/.local/share/virtualenvs/asgard-api-YwspFh3W/bin/python3
cachedir: .cache
rootdir: /home/daltonmatos/src/asgard-api, inifile: 
plugins: cov-2.4.0
collected 2 items 

tests/test_aiohttp_redirect_bug.py::AIOResponseRedirectTest::test_aiohttp_follows_mocked_redirect FAILED
tests/test_aiohttp_redirect_bug.py::AIOResponseRedirectTest::test_aiohttp_follows_passthrough_redirect PASSED

========================================================================================== FAILURES ===========================================================================================
________________________________________________________________ AIOResponseRedirectTest.test_aiohttp_follows_mocked_redirect _________________________________________________________________

self = <tests.test_aiohttp_redirect_bug.AIOResponseRedirectTest testMethod=test_aiohttp_follows_mocked_redirect>

    async def test_aiohttp_follows_mocked_redirect(self):
        with aioresponses() as rsps:
            rsps.get(
                "http://10.1.1.1:8080/redirect",
                status=307,
                headers={"Location": "https://httpbin.org"},
            )
            client = ClientSession()
            response = await client.get(
                "http://10.1.1.1:8080/redirect", allow_redirects=True
            )
>           self.assertEqual(200, response.status)
E           AssertionError: 200 != 307

tests/test_aiohttp_redirect_bug.py:18: AssertionError
============================================================================= 1 failed, 1 passed in 1.18 seconds ==============================================================================

```

Shouldn't the failed test return a `status=200` with the `history` attribute containing the redirected request? Is there anything that I'm doing wrong?

The versions are:

```
$ pipenv run pip freeze | grep -E "aiohttp|aioresp"
Loading .env environment variables…
aiohttp==3.4.4
aiohttp-cors==0.7.0
aioresponses==0.6.0
```

Python is `3.6.8`:

```
$ pipenv run python -V
Loading .env environment variables…
Python 3.6.8
```

Thanks,

**Comment by daltonmatos** [comment]
Hello @pnuckowski,

now that this issue is solved (by #138) are there any plans to release a new versoin of aioresponses?

thanks,

**Comment by asfaltboy** [comment]
@daltonmatos in case you haven't noticed (I haven't!) this was released a few weeks back in [version 0.6.3][1] which is now [available through PyPi][2].

[1]: https://github.com/pnuckowski/aioresponses/compare/0.6.2...0.6.3
[2]: https://pypi.org/project/aioresponses/

**Comment by daltonmatos** [comment]
Thanks @asfaltboy !!
